modules/rosapps/rosapps_shortcuts.inf

```diff
@@ -1,405 +1,405 @@
-[Version]
```

Comment:
Why has this file completely been changed?
Be sure its encoding remains **UTF-8 without BOM**

Reply:
I don't know what happened. Maybe my text editor changed the encoding.

Reply:
I'd recommend using Notepad++ as text editor. Also it's good practice to test your changes by building ReactOS and running it in virtual machine to see how your changes are looking.

Reply:
I used this, but I'm using it in Debian, and I think I have configurated with a wrong encoding. Sorry about that. 
I'll try to revise it before making a PR.

Reply:
In principle Npp is capable of recognizing the current encoding and keep it.